applications/nrf5340_audio/src/bluetooth/bt_stream/bt_le_audio_tx/bt_le_audio_tx.c

```diff
@@ -127,6 +128,54 @@ static int iso_stream_send(uint8_t const *const data, size_t size, struct bt_bap
 	return 0;
 }
 
+static int get_tx_sync_non_bt_ll_acs_nrf53(struct bt_bap_stream *stream,
+					   struct bt_iso_tx_info *info)
+{
+	struct net_buf *buf;
+	struct net_buf *rsp;
+	sdc_hci_cmd_vs_iso_read_tx_timestamp_t *cmd_read_tx_timestamp;
+	uint16_t conn_handle;
+
+	int err = bt_hci_get_conn_handle(stream->conn, &conn_handle);
+
+	if (err) {
+		LOG_ERR("Failed obtaining conn_handle (err:%d)", err);
+		return err;
+	}
```

Comment:
Nit:
```suggestion
	if (ret) {
		LOG_ERR("Failed obtaining conn_handle (err:%d)", ret);
		return ret;
	}
```
We use `ret` instead of `err`, and we usually declare it as the first variable in the function

Reply:
Did you change it back to using err?